components/help-modal.tsx

```diff
@@ -72,7 +78,9 @@ export default function HelpModal({ show = false, onCloseClick }: { show: boolea
         title={String(t("modals.help-modal.title"))}
         titlePosition="center"
         onCloseClick={onCloseClick}
-        backdrop={true} >
+        backdrop
+        fullscreen={isMobile as string | true}
```

Comment:
since all modals will be fullscreen when on mobile, we can simply add this to the modal.tsx component instead